### DISCUSSION TITLE
Welcome to -Group-1-Project-Management Discussions!

### DISCUSSION THREAD

**Discussion by MMikhailova** [lab-brussels-1/Group-1-Project-Management#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Corporatejoa** [comment]
Hello, I'm an aspiring tech project manager, I've managed few project from scratch to launch outside tech but I'm very much new to the tech industry. Will appreciate if you can share ideas and advice on how to start up This journey. Looking forward to hearing from you all.
Corporate joa
